handbook/sales/records.md

```diff
@@ -5,4 +5,8 @@ The [source of truth for our communications with customers is Salesforce](../com
 - All emails interacting with a customer must be logged to Salesforce (either by using [Einstein Activity Capture](https://help.salesforce.com/articleView?id=einstein_sales_aac.htm&type=5) or by manually adding the email to the relevant object). To set up Einstein Activity Capture you can [read this article](https://help.salesforce.com/articleView?id=aac_enable.htm&type=5) or [watch this video](https://www.youtube.com/watch?v=yVO9XnsW2vA).
 - Notes from all phone calls with customers should be logged in Salesforce. The call notes should be logged to the Opportunity, rather than the Contact or Account.
 
-For non-Sales team members who want to log emails without fully connecting via Einstein Activity Capture, set up an [Email to Salesforce](https://help.salesforce.com/articleView?id=email_my_email_2_sfdc_setup.htm&type=5) email address and BCC it on client emails.
+For non-Sales team members should also log communication with customers. In particular, the product team should BCC both Salesforce and HubSpot on emails, and log other forms of communication (such as a call) in Salesforce. 
+- If you have a Salesforce account who want to log emails without fully connecting via Einstein Activity Capture, set up an [Email to Salesforce](https://help.salesforce.com/articleView?id=email_my_email_2_sfdc_setup.htm&type=5) email address and BCC it on client emails.
```

Comment:
```suggestion
- If you have a Salesforce account, set up an [Email to Salesforce](https://help.salesforce.com/articleView?id=email_my_email_2_sfdc_setup.htm&type=5) email address and BCC it on client emails. Alternatively, you can fully connect your email via Einstein Activity Capture, but this is much more involved and not encouraged for Product team members.
```

Reply:
Apologies, I'm still not familiar with reviewing in Github, so hopefully I'm adding this comment in an appropriate place :)

Do we want to catalogue / make available a Salesforce BCC for non-SFDC users?  Maybe not make available as the handbook is public, but offer instructions for people to reach out to XXXX for a BCC and that person can add the person's email to acceptable email addresses and provide their BCC? 

Since the conversation yesterday I've been googling if there are good ways to handle this, but I haven't been able to give it too much time yet.

Reply:
I think that's what we're basically doing with Christina's account, but if you want to centralize this process outside of her account then go for it!

Reply:
On re-read you are completely correct, I was distracted by the pretty colors and not the very next line.